qa/tasks/cephadm.py

```diff
@@ -1992,14 +2002,35 @@ def configure_samba_client_container(ctx, config):
         '-eKRB5_CONFIG=/dev/null',
         samba_client_image,
     ]
-
     setattr(ctx, 'samba_client_container_cmd', samba_client_container_cmd)
     try:
         yield
     finally:
         setattr(ctx, 'samba_client_container_cmd', None)
 
 
+@contextlib.contextmanager
+def configure_samba_toolbox_container(ctx, config):
```

Comment:
Given that the client image and toolbox image are so close, why not just combine this with the configure_samba_client_container func? Always get both images at once. Call it `configure_samba_client_containers` or something? (note the trailing `s`).